styles.css

```diff
@@ -113,33 +114,39 @@ textarea {
 
 input {
   padding: 10px;
+  border-radius: 4px;
 }
 
 textarea {
   height: 100px;
   overflow: scroll;
+  border-radius: 4px;
 }
 
 .save {
   color: #BFBFDD;
-  padding: 16px;
+  padding: 12px;
   font-size: 24px;
   font-family: 'Open sans';
   font-weight: bold;
+  border-radius: 4px;
 }
 
 .disable-save-btn {
   background-color: #585759;
 }
 
-.active-save-btn {
+.save.active-save-btn {
```

Comment:
Nice job combining these two classes for greater specificity! 🔥

Reply:
That was harder than it should've been, haha.